src/coreComponent/keyboard.js

```diff
@@ -188,3 +188,13 @@ export function isVerticalMovementKey(keyCode) {
 export function isHorizontalMovementKey(keyCode) {
   return [KEY.LEFT, KEY.RIGHT, KEY.BACKSPACE, KEY.DELETE].includes(keyCode);
 }
+
+/**
+ * Returns true if pressing this key should select a highlighted item.
+ *
+ * @param {Number} keyCode
+ * @returns {Boolean}
+ */
+export function isSelectKey(keyCode) {
+  return [KEY.ENTER, KEY.TAB].includes(keyCode);
```

Comment:
`includes` is not supported in IE11. I don't think we should include a polyfill in this lib either -- that would just result in double polyfill code in our actual app.

```
keyCode === KEY.ENTER || keyCode === KEY.TAB
```

is not that bad and is probably faster, anyway.